posthog/hogql/database/schema/persons.py

```diff
@@ -30,14 +29,30 @@
 
 
 def select_from_persons_table(requested_fields: Dict[str, List[str]]):
-    return argmax_select(
-        table_name="raw_persons",
-        select_fields=requested_fields,
-        group_fields=["id"],
-        argmax_field="version",
-        deleted_field="is_deleted",
+    from posthog.hogql.parser import parse_select
+    from posthog.hogql import ast
+
+    query = parse_select(
+        """
+        SELECT id FROM raw_persons WHERE (id, version) IN (
```

Comment:
What happens with this id? Does it get replaced with the actual fields we need (properties, created_at) or is this going into another subquery?

Reply:
look a few lines lower